package.json

```diff
@@ -338,6 +338,16 @@
           "command": "mssql.showExecutionPlanInResults",
           "when": "editorLangId == sql && config.mssql.enableRichExperiences",
           "group": "navigation@4"
+        },
+        {
+          "command": "mssql.enableActualPlan",
+          "when": "editorLangId == sql && config.mssql.enableRichExperiences && !(resource in mssql.executionPlan.urisWithActualPlanEnabled)",
```

Comment:
this condition seems a little specific, but I guess we need it "resource in mssql.executionPlan.urisWithActualPlanEnabled"?

Reply:
it's to make sure we know which queries have the plan enabled  if there are multiple open

Reply:
if only json supported comments 🙃